<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Freelance Navigator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="upworkTab">
       <attribute name="title">
        <string>Upwork</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,10">
        <item>
         <layout class="QVBoxLayout" name="upworkFiltersLayout">
          <item>
           <widget class="QComboBox" name="upworkCategoryComboBox"/>
          </item>
          <item>
           <widget class="QListWidget" name="upworkSubcategoryListWidget">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Posted Date:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="upworkPostedDateComboBox">
              <item>
               <property name="text">
                <string>Any</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Within 24 hours</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Within 3 days</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Within 5 days</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Within a week</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Within 10 days</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Within 2 weeks</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Within a month</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Job Type:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="upworkJobTypeComboBox">
              <item>
               <property name="text">
                <string>Any</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fixed Price</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hourly</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="upworkBudgetLabel">
              <property name="text">
               <string>Budget:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="upworkMinBudgetLabel">
                <property name="text">
                 <string>$</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="upworkMinBudgetLineEdit">
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="upworkMaxBudgetLabel">
                <property name="text">
                 <string>to $</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="upworkMaxBudgetLineEdit">
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>max</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="upworkSearchLineEdit">
            <property name="placeholderText">
             <string>Search query</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="upworkSearchButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="FreelanceNavigator::Widgets::JobListWidget" name="upworkJobListWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionUpworkLogIn"/>
    <addaction name="actionUpworkLogOut"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About &amp;Freelance Navigator...</string>
   </property>
  </action>
  <action name="actionUpworkLogIn">
   <property name="text">
    <string>Log in to Upwork</string>
   </property>
  </action>
  <action name="actionUpworkLogOut">
   <property name="text">
    <string>Log out from Upwork</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FreelanceNavigator::Widgets::JobListWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/joblistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
